spec/index.bs

```diff
@@ -3892,18 +3830,84 @@ Issue: link to the semantics of SV_SampleIndex and SV_Coverage in WGSL spec.
         </div>
 </dl>
 
-### Primitive Topology ### {#primitive-topology}
+### Primitive Assembly ### {#primitive-assembly}
+
+<script type=idl>
+dictionary GPUPrimitiveAssemblyStateDescriptor {
+    GPUPrimitiveTopology topology = "triangle-list";
+    GPUIndexFormat stripIndexFormat;
+};
+</script>
+
+- {{GPUPrimitiveAssemblyStateDescriptor/topology}} defines what kind of
+    primitives are constructed by the [=pipeline=].
+- {{GPUPrimitiveAssemblyStateDescriptor/stripIndexFormat}} defines the supported
+    index format for strip topologies.
 
 <script type=idl>
 enum GPUPrimitiveTopology {
     "point-list",
     "line-list",
     "line-strip",
     "triangle-list",
-    "triangle-strip"
+    "triangle-strip",
+};
+</script>
+
+<div algorithm>
+    <dfn abstract-op>validating GPUPrimitiveAssemblyStateDescriptor</dfn>(|device|, |descriptor|)
+        1. If |device| is lost return `false`.
```

Comment:
This seems unnecessarily deep in the call stack. I'm sure we're inconsistent about this, but I think this should always be checked in the steps for each actual IDL method, and never or almost never in descriptor validation